dataflow-website/recipes/scaling/scdf-alert-webhook/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.dataflow.client.server-uri=http://192.168.99.100:30924
```

Comment:
Do we still want the URI hardcoded? Maybe this gets overridden by the deployment?